plugins/wallabag.koplugin/main.lua

```diff
@@ -700,6 +716,47 @@ Restart KOReader after editing the config file.]]), DataStorage:getSettingsDir()
     self.settings_dialog:onShowKeyboard()
 end
 
+
+function Wallabag:editClientSettings()
+    self.client_settings_dialog = MultiInputDialog:new {
+        title = _("Wallabag client settings"),
+        fields = {
+            {
+                text = self.articles_per_sync,
+                description = T(_("Articles to fetch:")),
+                input_type = "number",
+                hint = _("Articles to fetch")
+            },
+        },
+        buttons = {
+            {
+                {
+                    text = _("Cancel"),
+                    callback = function()
+                        self.client_settings_dialog:onClose()
+                        UIManager:close(self.client_settings_dialog)
+                    end
+                },
+                {
+                    text = _("Apply"),
```

Comment:
```suggestion
                    text = _("Set number"),
```

Reply:
I was wondering if `Apply` would be better, as I would like later on to add another setting to archive finished articles instead of deleting them (if the user wants to). Should we change it anyway and see to that potential new feature later?

Reply:
Alright, in that case we'd just cause unnecessary work for the translators.